kolibri/plugins/registry.py

```diff
@@ -48,14 +67,15 @@ def initialize():
             try:
                 plugin_module = importlib.import_module(app + ".kolibri_plugin")
                 logger.debug("Loaded kolibri plugin: {}".format(app))
+                all_classes = dict([(name, cls) for name, cls in plugin_module.__dict__.items() if isinstance(cls, type)])
                 plugin_classes = []
-                for obj in plugin_module.PLUGINS:
+                for obj in all_classes.values():
```

Comment:
hmmm, maybe you are right that something needs to be introspected further, like the `__module__` attribute matches.

Example:

```
>>> from kolibri.core import hooks
>>> hooks.NavigationHook.__module__
'kolibri.core.hooks'
>>> hooks.KolibriHook.__module__
'kolibri.plugins.hooks'
``` 

An imported class will not have a matching `__module__` property!